native/package.json

```diff
@@ -17,12 +17,12 @@
     ]
   },
   "devDependencies": {
-    "@storybook/react-native": "^3.2.11",
-    "babel-preset-react-native": "3.0.2",
-    "react-dom": "^16.0.0",
-    "react-test-renderer": "^16.0.0",
+    "babel-preset-react-native": "^4.0.0",
+    "@storybook/react-native": "^3.2.12",
     "react": "^16.0.0",
-    "react-native": "^0.47.0"
+    "react-dom": "^16.0.0",
+    "react-native": "^0.49.3",
```

Comment:
Just a quick one on this, will lock this down on the full version or we allow patch level update (0.49.*)?  there has been much discussion about this but no action yet

Reply:
Have a look at this: https://runkit.com/embed/umidffgbnot2

(The caret has different behaviour depending on pre `1.0.0`)